### ISSUE TITLE
Inquiry About the Hyper-Parameters for Replication

### ISSUE THREAD

**Issue by X1AOX1A** [Chaos96/fourierft#5]
### Inconsistent Hyper-Parameters Between Paper and Code

Hi there,

I found some inconsistencies in the hyper-parameters between the paper and the code. Here are the details:

1. **Issue with `n` for Roberta-base on GLUE:**
    - In Table 1, the value of `n` for Roberta-base is 200.
    - In Table 9, the value of `n` for both Roberta-base and Roberta-large is 1000.
    - In the [script](https://github.com/Chaos96/fourierft/blob/01101a2926b51c6777142cd7ff3d99344fc61efc/experiments/GLUE/cola-base-best.sh#L5C5-L5C25), `n` for Roberta-base is set to 1000.
    - Screenshots:
        - ![Table 1](https://github.com/Chaos96/fourierft/assets/52992366/776a7503-c76e-4725-96b7-435039a18167)
        - ![Table 9](https://github.com/Chaos96/fourierft/assets/52992366/89e697e2-3696-4677-bf30-5eb49360508b)

2. **Issue with LLaMA's Learning Rate and Scaling:**
    - In Table 11, the learning rate and scaling are set to 3E-3 and 16, respectively.
    - In the scripts:
        - The learning rate for [llama1-7B is 0.1](https://github.com/Chaos96/fourierft/blob/01101a2926b51c6777142cd7ff3d99344fc61efc/experiments/Instruction_Tuning/llama-1-7b.sh#L7)
        - The learning rate for [llama1-13B](https://github.com/Chaos96/fourierft/blob/01101a2926b51c6777142cd7ff3d99344fc61efc/experiments/Instruction_Tuning/llama-1-13b.sh#L7), [llama2-7B](https://github.com/Chaos96/fourierft/blob/01101a2926b51c6777142cd7ff3d99344fc61efc/experiments/Instruction_Tuning/llama-2-7b.sh#L7), and [llama2-13B](https://github.com/Chaos96/fourierft/blob/01101a2926b51c6777142cd7ff3d99344fc61efc/experiments/Instruction_Tuning/llama-2-13b.sh#L7) is 0.05.
        - The scaling is set to 300.
    - Screenshot:
        - ![Table 11](https://github.com/Chaos96/fourierft/assets/52992366/7dd4f72d-c2e4-42cf-b24f-6ceb97af4864)

Could you please clarify the correct hyper-parameters to replicate the results from the paper?

Thank you!
